website/static/js/logTextParser.js

```diff
@@ -72,7 +72,7 @@ var returnTextParams = function (param, text, logObject) {
 };
 
 var LogText = {
-    view : function(ctrl, logObject) {
+    view : function(ctrl, logObject, trackingLocation, action) {
```

Comment:
See https://github.com/CenterForOpenScience/osf.io/pull/5177/files#r56344445 . Note: you may need to rename `action` to avoid clashing with `logObject.action`.

```js
var options = $.extend({}, logObject, {trackingLocation: ..., trackingAction: ...});
```